packages/snowpack/src/commands/build.ts

```diff
@@ -24,14 +26,13 @@ import {printStats} from '../stats-formatter';
 import {CommandOptions, SnowpackSourceFile} from '../types/snowpack';
 import {getEncodingType, getExt, replaceExt} from '../util';
 import {getInstallTargets, run as installRunner} from './install';
-import {paint} from './paint';
 
 async function installOptimizedDependencies(
   allFilesToResolveImports: Record<string, SnowpackSourceFile>,
   installDest: string,
   commandOptions: CommandOptions,
 ) {
-  console.log(colors.yellow('! optimizing dependencies...'));
+  console.log(colors.yellow('! installing dependencies...'));
```

Comment:
the idea here was really "installing optimized dependencies", with tree-shaking and everything. Happy with either for now, but lets also throw up a reminder to discuss terminology for "install", post-2.7.

Reply:
It’s mostly in context, it’s weird to see `optimizing dependencies ...` followed by `install complete`. It felt more consistent to see `installing…` / `install complete`. I was thinking if we’re marking when a step begins and ends, it’s probably best to use the same verb? Whatever that is.